include/mongoose.hrl

```diff
@@ -34,3 +34,11 @@
 %% ---------------------------------
 %% Logging mechanism
 -include("mongoose_logger.hrl").
+
+%% for worker_pool
+-record(mongoose_worker_pool, {name :: atom(),
+    selection_strategy :: atom(),
+    extra :: term(),
```

Comment:
Can this type be more specific?

Reply:
it used to be, but Michał requested this change :)

Reply:
Right, previously there was `path_prefix` field which was specific to http pools also, and used by them only. I suggested to rename it to sth generic which may or may not be used by other pools.

Reply:
OK, I assumed it will always be a list or a map. Fine then. :)